Docs/Product/STT_Module_vNext_PRD.md

```
@@ -0,0 +1,118 @@
+**STT Module vNext / TODO PRD (Draft)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)